Robot2018/src/org/usfirst/frc/team199/Robot2018/subsystems/IntakeEject.java

```diff
@@ -123,6 +124,40 @@ public void lowerIntake() {
 		rightVerticalSolenoid.set(rightSet);
 	}
 
+	/**
+	 * Toggles the left intake between open and closed
+	 */
+	public void toggleLeftIntake() {
+		boolean open = SmartDashboard.getBoolean("Left Horizontal Solenoid Open", true);
```

Comment:
This will cause toggleLeftIntake() to fail to toggle the intake if the operator has changed the value in SmartDashboard.

Instead of usinig SmartDashboard to maintain this state, consider maintaining it in IntakeEject.java and just *displaying* it on the SmartDashboard. To allow the state to be modified via SmartDashboard, add the IntakeEject commands to SmartDashboard as described here: https://wpilib.screenstepslive.com/s/currentCS/m/shuffleboard/l/831050-working-with-commands-and-subsystems

Reply:
Addressed in latest commit.